tn_api_spec.yml

```diff
@@ -779,16 +775,116 @@ components:
             maxLength: 2
             description: "Airline carrier of the flight."
             example: "AC"
+          flight_time:
+            type: integer
+            description: Flight duration in minutes
+            example: 475
+          carrier_fullname:
+            type: string
+            description: Full name of the carrier
+            example: Air Canada
```

Comment:
String?

Reply:
Will change the values in the places i touched, this is another good candidate for an across the board refactor as-well!